src/css/common/theme.scss

```diff
@@ -0,0 +1,57 @@
+/*
+  COLORS
+ */
+
+$primary-font-color: #000;
+$secondary-font-color: rgb(136, 136, 136);
+
+$icon-button-color: rgb(59, 67, 71);
+$icon-button-color--selected: #2b9344;
+
+$separator-line-color: rgb(210, 212, 210);
+
+$additional-info-color: #5679B0;
+$clarification-color: #FDE300;
+$error-color: #F7AB00;
+
+$demagog-color: rgb(255,59,0);
+
+/*
+  FONTS
+ */
+
+$open-sans: "Open Sans", sans-serif;
+
+$normal-font-size: 14px;
+$secondary-font-size: 12px;
+$caption-font-size: 10px;
+
+$font-light: 300;
+$font-semi-bold: 600;
+
```

Comment:
We currently divide variables into fonts, colours etc. while this is a "theme-wise" file partitioning. 
I'm not sure what would be the best policy ("element-wise" or "theme-wise"), but we had better settle for one and use it consistently throughout the application.
Any thoughts on this @glowka , @lgodziejewski ? (I have never used SCSS/LESS long enough to have an opinion on that)
I think it's reasonable to settle for one option before merging.